.prettierrc

```diff
@@ -1,6 +1,13 @@
 {
-  "semi": true,
-  "trailingComma": "all",
+  "arrowParens": "avoid",
+  "bracketSameLine": true,
+  "bracketSpacing": true,
   "singleQuote": true,
-  "printWidth": 100
-}
+  "jsxSingleQuote": false,
+  "trailingComma": "all",
+  "semi": true,
+  "printWidth": 120,
```

Comment:
why did you extend width of line?

Reply:
Yes I did it to keep consistency with the backend and the web app